CorsixTH/Lua/app.lua

```diff
@@ -671,10 +672,28 @@ function App:getAbsolutePathToLevelFile(level)
   return nil, "Level not found: " .. level
 end
 
--- Loads the specified level. If a string is passed it looks for the file with the same name
+--! Invokes a protected call of App:_loadLevel(...). See that function for more information.
+--! This function should always be called to catch errors and properly pass the
+--! error to the player
+--!param error_prefix (string) Prefixes the error relevant to what was loaded
+--! return (boolean) The outcome of the pcall
+function App:loadLevel(error_prefix, level, difficulty, level_name, level_file, level_intro, map_editor)
```

Comment:
error_prefix should not be the first argument as it's an optional and not central to the function purpose.

Reply:
I agree, I was being lazy.

Reply:
Moved to the end of the function params, but this may have been just as bad.